terraform/modules/indexer/rds.tf

```diff
@@ -0,0 +1,58 @@
+resource "aws_db_subnet_group" "indexer" {
+  name       = "indexer"
+  subnet_ids = var.subnet_ids
+
+  tags = local.default_tags
+}
+
+resource "aws_security_group" "indexer" {
+  name   = "indexer-${local.workspace_name}"
+  vpc_id = var.vpc_id
+  tags   = local.default_tags
+
+  ingress {
+    from_port   = 5432
+    to_port     = 5432
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+}
+
+
+resource "aws_db_instance" "indexer" {
+  identifier             = "indexer-${local.workspace_name}"
+  instance_class         = "db.t3.micro"
+  allocated_storage      = 5
```

Comment:
Is this the size of the storage? can you make this configurable? also the instance class